src/constants/assetsConstants.js

```diff
@@ -75,6 +75,11 @@ const tBTC = 'tBTC';
 const OBTC = 'OBTC';
 const wBTC = 'wBTC'; // testnet
 const testBTC = 'testBTC'; // testnet
+const MATIC = 'MATIC';
```

Comment:
Please merge with develop, as it already contains MATIC & XDAI constants that I have added recently.